sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -157,15 +157,19 @@ def test_convergence_warning():
     assert_no_warnings(mdl.fit, X, y)
 
 
-def test_label_propagation_non_zero_normalizer():
+@pytest.mark.parametrize("label_propagation_class",
+                         [label_propagation.LabelSpreading,
+                          label_propagation.LabelPropagation])
+def test_label_propagation_non_zero_normalizer(label_propagation_class):
     # check that we don't divide by zero in case of null normalizer
     # non-regression test for
     # https://github.com/scikit-learn/scikit-learn/pull/15946
+    # https://github.com/scikit-learn/scikit-learn/pull/19271
```

Comment:
I think linking to the issue would be a little better in this case. The PR will be linked in the git commit when merged.

```suggestion
    # https://github.com/scikit-learn/scikit-learn/issues/9292
```